.gitignore

```diff
@@ -69,3 +69,8 @@ ec2.py
 
 hypermap/media
 hypermap/settings/local_settings.py
+
+# IDE specific
+/hypermap/settings/_panchicore.py
```

Comment:
@panchicore is a crazy colombian developer who is working on scaling. I don't think we need this change in the long run.